### ISSUE TITLE
Always returning same predicted label

### ISSUE THREAD

**Issue by gcornella** [sagieppel/Classification-of-object-in-a-specific-image-region-using-a-convolutional-neural-net-with-ROI-mask-a#2]
Hi, 
I am using your code, and I am always getting the same Predicted Label -> surfboard for all the test images that you propose.
Are you aware of this issue?
Also, in 'Resnet50Attention.py' you are defining RGBMean = [123.68, 116.779, 103.939] and RGBStd = [65, 65, 65], nevertheless, these values do not change when choosing a different test image.
Thank you,

**Comment by sagieppel** [comment]
The statistics  RGBMean, RGBStd is for images in generals and not for a specific image.
Are you using the pretrained model?

**Comment by gcornella** [comment]
Yes, I am just cloning your repo and running 'RunbPrediction.py'  with the files that you provide:

Trained_model_path="logs/WeightsRegionSpecificClassification.torch" # Pretrained model
ImageFile='TestImages/Test4/Image.png' #Input image
ROIMaskFile= 'TestImages/Test4/InputMask4.png' # Input ROI mas

However, I always get...
Predicted Label surfboard
Predicted Label Prob=1.59737728536129%

I will try cloning the repo again, but I believe I am following the steps properly

**Comment by sagieppel** [comment]
You're right, that is very strange.

**Comment by sagieppel** [comment]
Ok solved, when you read the image it needs to have a value of 0-255.
The image reader I used had its behavior changed so the pixel values were 0-1 and that confused the net.
I change the reader into cv2 readers which have pixel values 0-255, and that completely solved the problem.
Thanks for letting me know.
